### PR TITLE
Release 12.15.8

### PR DESCRIPTION
**Pull request by srenatus**
# [Release TODO](https://github.com/chef/chef-server/blob/master/RELEASE_PROCESS.md):
- [x] Merge #1311
- [X] inform #cft-announce and #chef-server
- [x] whitness pool/qa-chef-server-cluster passing
- [x] [Update Changelog](https://github.com/chef/chef-server/blob/master/RELEASE_PROCESS.md#preparing-for-the-release)
- [x] ⚠️ Clear acceptance (POOL-589 and POOL-590 need to be signed-off)
- [ ] [Tag, trigger, promote](https://github.com/chef/chef-server/blob/master/RELEASE_PROCESS.md#building-and-releasing-the-release)
- [ ] Discourse post

### CODE REVIEW

**Comment by afiune** [comment]
Qa chef-server cluster tests passed! Merged PR #1311

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 # Chef Server Changelog
 
+## [#](https://github.com/chef/chef-server/tree/#) (2017-06-19)
```
Comment (afiune) [inline CHANGELOG.md]: Prob match the same date we are trying to release, the 20th that is.

Comment (afiune) [inline CHANGELOG.md]: And the version 12.15.8.

**Comment by Sgtpluck** [comment]
@chef/chef-server-maintainers pls review!

**Pull request merged**